src/components/DocumentationTopic.vue

```diff
@@ -56,9 +56,25 @@
           v-if="shouldShowAvailability"
           :platforms="platforms" :technologies="technologies"
         />
+        <div
+          v-if="declarations.length"
+          class="declarations-container"
+          :class="{ 'minimized-container': enableMinimized }"
+        >
+          <Declaration
+            v-for="(declaration, index) in declarations"
```

Comment:
Would this ever actually loop over more than a single "declarations" section? I think even in the case of a symbol with multiple platform-specific declaration variants, it's still all contained within the single primary content section, right?

Reply:
I think that is the case yes, I was just worried that if I treat it as a single entity I might break some weird edge case 🤣 

If you think its safe, I can remove the v-for :)

Reply:
I think it's safe. But it's also ok as-is.